easybuild/easyconfigs/i/Instant/Instant-1.5.0-goolf-1.5.14-Python-2.7.8.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = "PythonPackage"
+
+name = 'Instant'
+version = '1.5.0'
+
+homepage = 'https://launchpad.net/instant'
+description = """Instant is a Python module that allows for instant inlining of 
+ C and C++ code in Python. It is a small Python module built on top of SWIG and Distutils.
+ It is part of the FEniCS Project (http://fenicsproject.org)."""
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+
+#download from https://bitbucket.org/fenics-project/instant/downloads/
+
+source_urls = ['']
+sources = ['']
+
+python = "Python"
+pythonversion = '2.7.8'
```

Comment:
We'll have this problem on many eb's ... 
We use python 2.7.8, and at the moment we won't make the update yet. Not sure if you want me to have different sets of eb's, the ones that we use, and eb's for PRs? This would involve double testing as well. 
What is the EB idea in general?